modules/services/fusuma.nix

```diff
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+let cfg = config.services.fusuma;
+in with lib; {
+  meta.maintainers = with maintainers; [ iosmanthus ];
+  options.services.fusuma = {
+    enable = mkEnableOption
+      "the fusuma systemd service to automatically enable touchpad gesture";
+    package = mkOption {
+      type = types.package;
+      default = pkgs.fusuma;
+      defaultText = literalExpression "pkgs.fusuma";
+      description = "Package providing <command>fusuma</command>.";
+    };
+    settings = mkOption {
+      type = types.attrs;
```

Comment:
The `attrs` type should not be used except in very specific cases. I think you want to use [`pkgs.formats.yaml`](https://nixos.org/manual/nixos/stable/#sec-settings-nix-representable).